pkg/assets/rte/rtenotifier.template

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -e
+
+bundle=$(jq '.bundle' /dev/stdin 2>&1)
+pod_cgroup=$(cat ${bundle}/config.json | jq .linux.cgroupsPath)
+
+# Do not touch file for non guaranteed pods
+if [[ ${pod_cgroup} ~= ".*burstable.*" ]] || [[ ${pod_cgroup} ~= ".*besteffort.*" ]]; then
+    exit 0
+fi
+
+notify_file_dir=$(dirname {{.notifierFilePath}})
```

Comment:
this is nice, but I'd just try to touch and if the path doesn't exist, good riddance:
```
touch {{ .notifierFilepath }} || :
```